cluster-provision/k8s/scripts/provision.sh

```diff
@@ -144,8 +147,78 @@ $reset_command
 # audit log configuration
 mkdir /etc/kubernetes/audit
 
-# New configuration for kubernetes >= 1.12
-if [[ ${BASH_REMATCH[1]} -ge "12" ]]; then
+if [[ ${BASH_REMATCH[1]} -ge "15" ]]; then
+cat > /etc/kubernetes/audit/adv-audit.yaml <<EOF
```

Comment:
if I see correct, we have the same audit config file 3 times now. Can move that into a single `if [[ ${BASH_REMATCH[1]} -ge "11" ]]; then ... fi` block please?